src/Chunk.h

```diff
@@ -439,7 +439,12 @@ class cChunk :
 	void SetAlwaysTicked(bool a_AlwaysTicked);
 
 	// Makes a copy of the list
-	cClientHandleList GetAllClients(void) const {return m_LoadedByClient; }
+	cClientHandleList GetAllClients(void) const
+	{
+		cClientHandleList copy;
+		std::copy(m_LoadedByClient.begin(), m_LoadedByClient.end(), std::back_inserter(copy));
+		return copy;
```

Comment:
The difference is that using the copy constructor only works if you're returning the same type of list. In order to restrict the scope of this change, I didn't expose the change to a vector publicly. Which means it needs std::copy to copy from a vector to a list.